### DISCUSSION TITLE
Is there a way to collapse cell output?

### DISCUSSION THREAD

**Discussion by ForBo7** [quarto-dev/quarto-cli#5332] [Q&A] [answered]
Hello!

So I'm aware I can collapse a code ~~output~~ cell with the following cell option: `#| code-fold: true`.

However, does a cell option exist to collapse cell output instead of a code ~~output~~ cell?

**Comment by cscheid** [comment, marked as answer]
(I think you mean "input" in the first two instances of "output" :))

No, that's not an option yet, though we're going to try to get it done for 1.4.

**Reply by ForBo7** [reply]
Ah, okay. That's good to know.

**Comment by mcanouil** [comment]
To note, the related opened issue to track this #341
